pkg/plugins/policies/imports.go

```diff
@@ -1,18 +1,44 @@
 package policies
 
 import (
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshaccesslog"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshcircuitbreaker"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshfaultinjection"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshhealthcheck"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshhttproute"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshloadbalancingstrategy"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshmetric"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshproxypatch"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshratelimit"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshretry"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshtcproute"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshtimeout"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshtrace"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshaccesslog"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshcircuitbreaker"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshfaultinjection"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshhealthcheck"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshhttproute"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshloadbalancingstrategy"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshmetric"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshproxypatch"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshratelimit"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshretry"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshtcproute"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshtimeout"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshtrace"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission"
 )
+
+var nameToModule = map[string]func(bool){
+	"meshaccesslog":             meshaccesslog.InitPlugin,
+	"meshcircuitbreaker":        meshcircuitbreaker.InitPlugin,
+	"meshfaultinjection":        meshfaultinjection.InitPlugin,
+	"meshhealthcheck":           meshhealthcheck.InitPlugin,
+	"meshhttproute":             meshhttproute.InitPlugin,
+	"meshloadbalancingstrategy": meshloadbalancingstrategy.InitPlugin,
+	"meshmetric":                meshmetric.InitPlugin,
+	"meshproxypatch":            meshproxypatch.InitPlugin,
+	"meshratelimit":             meshratelimit.InitPlugin,
+	"meshretry":                 meshretry.InitPlugin,
+	"meshtcproute":              meshtcproute.InitPlugin,
+	"meshtimeout":               meshtimeout.InitPlugin,
+	"meshtrace":                 meshtrace.InitPlugin,
+	"meshtrafficpermission":     meshtrafficpermission.InitPlugin,
+}
+
+func InitPolicies(enabledPluginPolicies []string) {
```

Comment:
If we change this to `func init()` does it mean the existing imports like ` _ "github.com/kumahq/kuma/pkg/plugins/policies"` will work?

Reply:
we can't use init unless we manually read the env variable responsible for handling switching on / off policies, do you think it's a better approach?

Reply:
I'm changing it now, we will see if it works better

Reply:
https://github.com/kumahq/kuma/pull/8828 created alternative PR with that approach